### ISSUE TITLE
Scenarios no longer work after most recent update

### ISSUE THREAD

**Issue by brute773** [M1lkm8n/jbad#34]
Hello,

I really appreciate all the work you've put into this mod. I have used it many times and am grateful that it is in the community. Within the last 6 months I used it in a few different scenarios found here,

https://steamcommunity.com/sharedfiles/filedetails/?id=2177457980

https://steamcommunity.com/sharedfiles/filedetails/?id=2183809275

Currently these scenarios no longer work and say they are missing content.

Players of my scenarios say that "afghan_houses" is missing. And when I try to open these scenarios up in the editor it tells me that

Misc_Steps
tv_a
kitchen_table_a
afghan_houses_c
afghan_houses_old
Wall_L
afghan_houses

are all missing. I'm hoping I can receive some help and we can get my scenarios back up and running. Unfortunately I left the missions binarized so I am unable to open the files and remove the missing files.

![20210405_211118](https://user-images.githubusercontent.com/82009084/113650859-52ce9400-9667-11eb-99b2-26e19918c47e.jpg)

If it works, here is a photo of my screen. Thanks,

Brute

**Comment by M1lkm8n** [comment]
Yes unfortunately with the latest update we needed to add jbad_ tags to each of those required addons that were missing them. This was so that it would stop conflicting addon reports in the log files. Moving forward any more jbad updates will no longer mess up missions or terrains.   

I guess you have a couple option. Either rebuild your mission and add the jbad tags or you can use this legacy mod which will fix the issue without you having to do anything other than subscribe to that mod. https://steamcommunity.com/sharedfiles/filedetails/?id=2442475820

**Comment by lsvMoretti** [comment]
Any updates on this @M1lkm8n or @brute773 - I'm trying to load this up on Liberation. I've put jbad and others as client mods. I presume this is correct. The above legacy doesn't work loaded either.
